### PR TITLE
[Robust Speech Challenge] Add missing LR parameter

### PR DESCRIPTION
**Pull request by jonatasgrosman**
Some participants of the Robust Speech event complain about learning rate issues when training the model with the run_speech_recognition_ctc_bnb.py script. This problem is due to a missing LR parameter on the 8-bit optimizer initialization. This PR fixes this.

@patrickvonplaten

### CODE REVIEW

**Review by patrickvonplaten — APPROVED** [review]
Thanks a lot!
